syscalls.c

```diff
@@ -261,12 +261,45 @@ int syscalls_endthread(void *ustack)
 }
 
 
-int syscalls_usleep(void *ustack)
+int syscalls_nsleep(void *ustack)
 {
-	unsigned int us;
+	time_t *sec;
+	long int *nsec;
+	time_t start, sleep, stop, diff;
```

Comment:
maybe don't use `sleep` as a variable name as it's a c-standard-library symbol (thus highlighted here)

Reply:
Done, thx